cmd/argocd/commands/login.go

```diff
@@ -45,16 +45,26 @@ func NewLoginCommand(globalClientOpts *argocdclient.ClientOptions) *cobra.Comman
 		Use:   "login SERVER",
 		Short: "Log in to Argo CD",
 		Long:  "Log in to Argo CD",
+		Example: `# Login to Argo CD using a username and password
+argo login cd.argoproj.io
```

Comment:
Should this be `argocd login` instead of `argo login`?

Reply:
good catch. Thank you! fixed